resources/config.yml

```diff
@@ -1,7 +1,7 @@
 # Texter config file
 
-# set language (en_us, ja_jp or ru_ru)
-locale: en_us
+# set language (zh_cn, en_us, ja_jp or ru_ru)
+locale: zh_cn
```

Comment:
English is the official language, so do not change the default

please change as below
```yaml
# set language (en_us, ja_jp, ru_ru or zh_cn)
locale: en_us
```

Reply:
ok